src/components/form/dt-multi-select/dt-multi-select.js

```diff
@@ -309,6 +314,27 @@ export class DtMultiSelect extends HasOptionsList(DtFormBase) {
     );
   }
 
+  _validateRequired() {
+    const { value } = this;
+
+    if (this.required && value) {
```

Comment:
If value is null/undefined and required is true, this won't trigger a required message